ConfigurationSystem/Client/Utilities.py

```diff
@@ -273,6 +273,13 @@ def addToChangeSet( entry, changeSet ):
           newMaxCPUTime = queueInfo.get( 'GlueCEPolicyMaxCPUTime', '' )
           if newMaxCPUTime == "4" * len( newMaxCPUTime ) or newMaxCPUTime == "9" * len( newMaxCPUTime ):
             newMaxCPUTime = ''
+          wallTime = queueInfo.get( 'GlueCEPolicyMaxWallClockTime', '' )
+          if wallTime and int(wallTime)>0:
+            if not newMaxCPUTime:
+              newMaxCPUTime = str(0.8*int(wallTime))
+            else:
+              if int(wallTime) <= int(newMaxCPUTime):
+                newMaxCPUTime = str(0.8*int(wallTime))
```

Comment:
If the wall time limit is shorter or equal to the CPU time limit, this is a bad idea from he site... Assume jobs have more than 80% CPU efficiency and get the CPU time limit hitting first